global.json

```diff
@@ -1,5 +1,5 @@
 {
     "msbuild-sdks": {
-        "Microsoft.Quantum.Sdk": "0.22.192435-beta"
+        "Microsoft.Quantum.Sdk": "0.22.192949-beta"
```

Comment:
Why not [0.22.192977](https://ms-quantum.visualstudio.com/Quantum%20Program/_build/results?buildId=192977&view=results)?

Reply:
This package number works perfectly fine, and was simply a complete set that was ready earlier.